pkg/cmd/convert.go

```diff
@@ -62,20 +69,44 @@ func GetValidatedObject(v *viper.Viper, cmd *cobra.Command, out, outerr io.Write
 	}
 
 	var ocObjects []*object.OpenCompose
+	var data []byte
+	var err error
 	for _, file := range files {
```

Comment:
I don't think it needs renaming. Especially as it's not changed here.  (Btw. Go guidelines actually tell you to use really short names, but to be fair it's no applicable everywhere.)